data/Ayan_khan.json

```
@@ -0,0 +1,23 @@
+{
+  "name": "Ayan Khan",
+  "type": "personal",
+  "bio": "explore things travelling  , 2nd year Cs student, web developer"
+}
+
+
+"links": [
+    {
+      "name": "Follow me on Instagram",
+      "url": "https://instagram.com/ayan_235__?igshid=YmMyMTA2M2Y=",
+      "icon": "FaInstagram"
+    }
+  ]
+}
+"links": [
+    {
+      "name": "Follow me on linkedIn",
```

a few corrections are required:
```suggestion
"bio": "explore things travelling  , 2nd year Cs student, web developer",
	"links": [{
		"name": "Follow me on Instagram",
		"url": "https://instagram.com/ayan_235__?igshid=YmMyMTA2M2Y=",
		"icon": "FaInstagram"
	}, {
		"name": "Follow me on linkedIn",
```